src/entities/tokens.ts

```diff
@@ -30,6 +30,14 @@ export interface ParsedAddressAsset {
   uniqueId: string;
 }
 
+export interface UniswapCurrency extends Asset {
```

Comment:
This will need to be fleshed out (and useUniswapAssets / useUniswapAssetsInWallet will need to be updated) but putting this in as a placeholder for now